Source/Controllers/SubviewControllers/SPTablesList.m

```diff
@@ -741,6 +741,24 @@ - (IBAction)togglePinTable:(nullable id)sender {
     // actual pin toggle will happen when notification is received and processed
 }
 
+
+- (IBAction)copyTableName:(nullable id)sender {
+    if (!selectedTableName) {
```

Comment:
Just a question - does it copy name of the "Selected" table or table that you have mouse over and right click on even in case it's not the selected one?

Reply:
right click will select the table first ( This is the default behavior of Sequel-Ace).

so, it copy the selected table's name.

Reply:
I wanted to copy the table mouse overed, but  I failed...